ui/types.go

```diff
@@ -0,0 +1,18 @@
+package ui
+
+// TextOpts Asking for text options
+type TextOpts struct {
+	Label   string
+	Default string
+	// ValidateFunc: method to validate input/default value
+	ValidateFunc func(string) (bool, error)
```

Comment:
should we have (bool, string, error) as return, since it would be wanted to let user know what is wrong with the input they entered if it's invalid.